src/components/Book.js

```diff
@@ -6,12 +6,44 @@ const Book = props => {
   } = props;
 
   return (
```

Comment:
## Optional Change
- [ ] You may need to destructure the following lines of code as it is making the `Book.js` component to be more verbose. The suggested destructing does achieve the same result with less code in a more reusable way. Take a look 👇 
- #### Current
```ruby
const Book = props => {
  const {
     bookId, bookTitle, bookCategory, remove,
  } = props;
```

- #### Suggestion 
```ruby
const Book = ({  bookId, bookTitle, bookCategory, remove }) => {
  ...
}
```

Reply:
Thank you for this suggestion. I've implemented it. Now the code looks better.